readme.md

```diff
@@ -0,0 +1,58 @@
+<div align="center">
+<img alt="Piston logo" src="/assets/90px-90px.png" width="100px" height="140px" />
+
+# PISTON UI
+</div>
+## Quick Start
+
+To start using the components in your project, Copy-paste the stylesheet `<link>` into your `<head>` before all other stylesheets to load our Styling to be used.
+
+```html
+    <link
+        rel="stylesheet"
+        href="https://piston-components.netlify.app/styles.css"
+    />
+```
+
+
+### [PISTON UI](https://piston-ui.netlify.app/index.html) contains the following components
+
+- [Alert](https://piston-ui.netlify.app/html-folder/alerts.html)
+- [Avatar](https://piston-ui.netlify.app/html-folder/avatars.html)
+- [Badge](https://piston-ui.netlify.app/html-folder/badges.html)
+- [Button](https://piston-ui.netlify.app/html-folder/buttons.html)
+- [Card](https://piston-ui.netlify.app/html-folder/cards.html)
+- [CSS Grid](https://piston-ui.netlify.app/html-folder/grids.html)
+- [Image](https://piston-ui.netlify.app/html-folder/image.html)
+- [Input](https://piston-ui.netlify.app/html-folder/inputs.html)
+- [List](https://piston-ui.netlify.app/html-folder/lists.html)
+- [Modal](https://piston-ui.netlify.app/html-folder/modal.html)
+- [Navigation]https://piston-ui.netlify.app/html-folder/nav.html)
```

Comment:
You missed one ( opening bracket here. Rest looks good. 👌

Reply:
Thanks Neha You missed one too here :)